### ISSUE TITLE
♿ Components meet AVT3 Requirements ♿

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1522]
**This epic tracks the work happening on Carbon Components toward AVT3 compliance.** Each component will go through user testing by [Team Able](https://www.ibm.com/blogs/age-and-ability/2018/03/07/ibm-employees-disabilities-volunteer-time-create-accessible-technology/) and other volunteer screen reader users and results will be shown below. 

Each component should be tested with JAWS 2018+, VoiceOver, and NVDA.

## Legend
🇳 User testing completed with NVDA
🇯 User testing completed with JAWS 2018+
🇻 User testing completed with VoiceOver
✅ The component passes AVT3 testing with no violations
❌ The component has failed AVT3 testing and issue(s) are linked

## Testing Environment
- [x] Build a staging/sandbox area for accessibility/usability testing a Carbon Components.
- [Carbon Accessibility Sandbox](https://carbon-accessibility-sandbox.netlify.com/)

## Component variations

- Accordion🇯❌[#1530](https://github.com/IBM/carbon-components/issues/1530)
- Breadcrumb🇯
- Button🇯
- Checkbox🇯❌ #1469 
- Code Snippet🇯❌ #1531
- Combo box🇯❌ #1532
- Content Switcher🇯❌ #1534
- Data Table🇯 ❌ #1578 #1579 #1580 
- Date Picker🇯 ❌ #1588 
- Dropdown 🇯 ❌ #1589 
- File Uploader 🇯 #1070 
- Footer🇯
- Form 🇯
- Grid🇯
- Link🇯
- List
- List Box 🇯
- Loading 🇯
- Modal 🇯
- Multi-select 🇯
- Notification 🇯
- Number Input 🇯
- Overflow Menu 🇯 ❌ #1615 #1233 
- Pagination 🇯 ❌ #1591 
- Progress Indicator 🇯
- Radio Button 🇯
- Search 🇯
- Select 🇯
- Skeleton🇯
- Slider 🇯
- Structured List 🇯
- Tabs 🇯 ❌ #1495
- Tag 🇯
- Text Area 🇯
- Text Input 🇯
- Tile 🇯
- Time Picker 🇯
- Toggle 🇯
- Toolbar🇯
- Tooltip 🇯 ❌ #1477 
- UI Shell 🇯

**Comment by dakahn** [comment]
Shutting this down. Plans to surface our testing results in a more readable and accessible way post-v10
